benchexec/tools/jlisa.py

```diff
@@ -0,0 +1,60 @@
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+import sys
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for JLiSA
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("jlisa")
+    
+    def version(self, executable):
+        return self._version_from_tool(executable, arg="version")
+
+    def name(self):
+        return "JLiSA"
+
+    def project_url(self):
+        return "https://github.com/lisa-analyzer/jlisa"
+
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable, "check", "--inputs", " ".join(task.input_files_or_empty)]
```

Comment:
Does your tool support tasks with more than one input file or without any input files? If not, please use `task.input_files` or `task.single_input_file` instead because these will produce a proper error message for the user if an unsupported task is used.